drivers/sensor/fxos8700/fxos8700_trigger.c

```diff
@@ -16,13 +16,13 @@ static void fxos8700_gpio_callback(const struct device *dev,
 {
 	struct fxos8700_data *data =
 		CONTAINER_OF(cb, struct fxos8700_data, gpio_cb);
+	const struct fxos8700_config *config = dev->config;
```

Comment:
`dev` is a GPIO device here, not a sensor device

Reply:
Fixed!

Reply:
Thank you!

Reply:
Well thank you, appreciate your thorough reviews!